insights/parsers/rpm_v_packages.py

```diff
@@ -58,3 +66,56 @@ def parse_content(self, content):
                 entry = {"line": line.strip(), "attributes": line_parts[0], "mark": None,
                          "file": line_parts[1]}
             self.packages_list.append(entry)
+
+
+@parser(Specs.rpm_V_package)
+class RpmVPackage(CommandParser):
+    """
+    Class for parsing ``/bin/rpm -V <package>`` command.
+
+    Attributes:
+        package_name (String): The name of package
+        package_list (list of dictionaries): Every dictionary contains information about one entry
```

Comment:
Since the new `RpmVPackage` is only for one single package, I don't think this attribute is suitable anymore.

How about renaming it as `discrepancies` according to the manual?
```
       rpm {-V|--verify} [select-options] [verify-options]

       Verifying a package compares information about the installed files in the package with information about the files taken from the package metadata stored in the rpm database.  Among other things, verifying compares the size, digest, permissions, type, owner and group of  each  file.
       Any discrepancies are displayed.  Files that were not installed from the package, for example, documentation files excluded on installation using the "--excludedocs" option, will be silently ignored.

       The package selection options are the same as for package querying (including package manifest files as arguments).  Other options unique to verify mode are:
```